tests/manage/monitoring/prometheusmetrics/test_ocs_utilization.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf8 -*-
+"""
+Test cases here performs Prometheus queries for non OCS metrics (such as cpu or
+memory utilization) of OCS components (such as ceph or noobaa) to find excesive
+resource usage.
+"""
+
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.utility.prometheus import PrometheusAPI
+from ocs_ci.utility.prometheus import check_query_range_result_limits
+
+
+logger = logging.getLogger(__name__)
+
+
+# cpu pod usage query inspired by Metrics Dashboard from OCP Console, see:
+# frontend/packages/dev-console/src/components/monitoring/queries.ts
+CPU_USAGE_POD = "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate"
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-2364")
+@pytest.mark.bugzilla('1849309')
```

Comment:
Use polarion_id and bugzilla marks from ocs_ci/framework/pytest_customization/marks.py

Reply:
fixed in 483b89cbc3a7506ad2db5ae47e7c98df488c8a8d